src/Wt/WResource.h

```diff
@@ -86,6 +86,10 @@ enum class ContentDisposition {
  * \endcode
  * \endif
  *
+ * In order to configure your web server to serve such a resource, either
+ * create a link to it using WAnchor or WImage, or add it to your WServer
+ * using WServer::handleRequest().
```

Comment:
I assume you mean `addResource`, not `handleRequest`.

Also: The "global and private resources" heading already makes mention of it. The only thing that's lacking is how to serve a private resource (i.e. through a `WLink`).